From: Jakub Adam <jakub.adam@ktknet.cz>
Date: Tue, 8 Nov 2011 22:54:49 +0100
Subject: debian-custom-build

- disable maven-source-plugin (we don't need jar archive with sources)
- disable findbugs-maven-plugin (not in Debian)
- disable maven-pmd-plugin (not in Debian)
- do not build org.eclipse.jgit.http.test (requires newer Jetty than present in Debian)
- add org.eclipse.jgit.pgm to jgit console classpath
- don't build jgit console all-in-one executable
---
 org.eclipse.jgit.pgm/jgit.sh |    2 +-
 pom.xml                      |    9 +++++----
 2 files changed, 6 insertions(+), 5 deletions(-)

--- a/org.eclipse.jgit.pgm/jgit.sh
+++ b/org.eclipse.jgit.pgm/jgit.sh
@@ -86,7 +86,7 @@
 	;;
 esac
 
-CLASSPATH="$cp"
+CLASSPATH="$cp:/usr/share/java/org.eclipse.jgit.pgm.jar"
 export CLASSPATH
 
 java=java
--- a/pom.xml
+++ b/pom.xml
@@ -400,6 +400,7 @@
           <links>
             <link>http://java.sun.com/j2se/1.5.0/docs/api</link>
           </links>
+          <detectOfflineLinks>false</detectOfflineLinks>
         </configuration>
         <executions>
           <execution>
@@ -517,7 +518,7 @@
       <modules>
         <module>org.eclipse.jgit.console</module>
         <module>org.eclipse.jgit.java7</module>
-        <module>org.eclipse.jgit.java7.test</module>
+        <!--module>org.eclipse.jgit.java7.test</module-->
       </modules>
     </profile>
     <profile>
@@ -604,11 +605,11 @@
     <module>org.eclipse.jgit.http.server</module>
     <module>org.eclipse.jgit.pgm</module>
     <module>org.eclipse.jgit.junit</module>
-    <module>org.eclipse.jgit.junit.http</module>
+    <!--module>org.eclipse.jgit.junit.http</module-->
 
-    <module>org.eclipse.jgit.test</module>
+    <!--module>org.eclipse.jgit.test</module-->
     <module>org.eclipse.jgit.ant.test</module>
-    <module>org.eclipse.jgit.http.test</module>
+    <!--module>org.eclipse.jgit.http.test</module-->
     <module>org.eclipse.jgit.pgm.test</module>
   </modules>
 
